prboom2/INSTALL

```diff
@@ -41,17 +17,15 @@ Basic Installation
        apt-get install libsdl2-dev libsdl2-net-dev libsdl2-image-dev \
          libpcre3-dev libsdl2-mixer-dev libfluidsynth-dev \
          libportmidi-dev libmad0-dev libdumb1-dev libvorbis-dev
-
+         
+    You will also need `cmake` and `make` if you haven't already got them.
+         
   2. `cd' to the directory containing the PrBoom distribution (the directory
-     this file is in).
+     this file is in, prboom2).
 
-     Type `./configure' to configure PrBoom for your system.  If you're
-     using `csh' on an old version of System V, you might need to type
-     `sh ./configure' instead to prevent `csh' from trying to execute
-     `configure' itself.
-
-     Running `configure' takes awhile.  While running, it prints some
-     messages telling which features it is checking for.
+     Run `cmake .` from this folder
```

Comment:
I'd recommend adding the `-DCMAKE_BUILD_TYPE=Release` flag to get a release build (cmake defaults to debug, which results in lower performance).

Reply:
I've made a change to this. I added a comment to say to remove it if you're wanting a debug build (I tend to run things as debug due to never quite being sure I've done everything right).